melvs/melvsboot/src/main/java/com/melvs/melvsboot/GetList.java

```diff
@@ -0,0 +1,41 @@
+package com.melvs.melvsboot;
+
+
+public class GetList {
+
+//string variable
+    private String id;
+    private String name;
+    private String description;
+
+//to initialize
+    public GetList(){
+
+    }
+
+    public GetList(String id,String name,String description){
+        super();
+        this.id = id;
+        this.name = name;
+        this.description = description;
+    }
```

Comment:
Still not sure what the purpose of this empty method. If you have time, please let me know. If not, I can just research on the topic.

Reply:
Empty constructor is there so you can do new GetList. Otherwise, you cannot create an instance without an id, name and description.